requirements.txt

```diff
@@ -1,6 +1,7 @@
 asgiref==3.2.10
 Django==3.1.2
-psycopg2-binary
+djangorestframework==3.12.2
+gunicorn==20.0.4
```

Comment:
@lealal Good idea to lock this down, do you know if this is the same version we are currently using?